providers/openlineage/docs/guides/user.rst

```diff
@@ -478,6 +478,51 @@ You can enable this automation by setting ``spark_inject_transport_info`` option
   AIRFLOW__OPENLINEAGE__SPARK_INJECT_TRANSPORT_INFO=true
 
 
+Passing parent information to Airflow DAG
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To enable full OpenLineage lineage tracking across dependent DAGs, you can pass parent and root job information
+through the DAG's ``dag_run.conf``. When a DAG run configuration includes an ``_openlineage`` section with valid metadata,
+this information is automatically parsed and converted into DAG's **ParentRunFacet**, from which the root information
+is also propagated to all task runs. If no DAG run configuration is provided, the DAG run itself is considered the
+lineage root for its tasks.
+
+The ``_openlineage`` dict in conf can contain the following keys:
+
+*(all three values must be included to create a parent reference)*
+
+- **parentRunId** — the unique run ID (uuid) of the direct parent job
+- **parentJobName** — the name of the parent job
+- **parentJobNamespace** — the namespace of the parent job
+
+*(all three values must be included to create a root reference)*
+
+- **rootParentRunId** — the run ID (uuid) of the top-level (root) job
+- **rootParentJobName** — the name of the top-level (root) job
+- **rootParentJobNamespace** — the namespace of the top-level (root) job
+
+
+Example:
+
+.. code-block:: python
+
+    conf = {
+        "_openlineage": {
+            "parentRunId": "3bb703d1-09c1-4a42-8da5-35a0b3216072",
+            "parentJobNamespace": "prod_biz",
+            "parentJobName": "get_files",
+            "rootParentRunId": "9d3b14f7-de91-40b6-aeef-e887e2c7673e",
+            "rootParentJobNamespace": "prod_analytics",
+            "rootParentJobName": "generate_report_sales_e2e",
+        }
+    }
+    trigger_dagrun = TriggerDagRunOperator(
+        task_id="trigger_dagrun",
+        trigger_dag_id="child_dag",
+        conf=conf,
+    )
```

Comment:
Can we not automatically set those?
Might require change to `TriggerDagRunOperator`, but there's nothing wrong with that AFAIK. And if OL is installed and running, all of this information should be available.